.github/workflows/performance.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         run: ./benchmark_output_to_json.sh output.txt output.json
 
       - name: 'Store benchmark result'
-        uses: benchmark-action/github-action-benchmark@v1
+        uses: benchmark-action/github-action-benchmark@v1.17.0
```

Comment:
How does this work when `v1.18.0` is released?

Reply:
Leaving `v1` targets to the latest version, so I'll suggest to keep it as it was